### PR TITLE
rollback-slam-vue-to-pre-map-representation

### PR DESCRIPTION
**Pull request by nicksanford**
1. Rolls back slam.vue to call GetMap (image/jpeg) for the 2d view rather than GetPointCloudMapStream
2. Removes 3d view
3. Changes 2d view to only allow polling at 5, 10, and 30 second intervals or manual refresh
4. Bump remote-control to version `0.2.4`

This leaves all the code for using the temp post map representation APIs unchanged so that we can roll forward when the new APIs have the required performance characteristics. 

Screenshot 2023-03-10 at 12 23 19 PM
Screenshot 2023-03-10 at 12 22 53 PM

### CODE REVIEW

File: web/frontend/src/components/slam.vue [hunk 1]
```diff
@@ -39,6 +40,21 @@ const concatArrayU8 = (arrays: Uint8Array[]) => {
...
+  props.client.slamService.getMap(req, new grpc.Metadata(), (error, response) => {
+    if (error) {
+      return displayError(error);
+    }
+    const blob = new Blob([response!.getImage_asU8()], { type: 'image/jpeg' });
+    imageMap = URL.createObjectURL(blob);
+  });
+};
+
 const fetchSLAMMap = (name: string) => {
```
Comment (tessavitabile) [inline web/frontend/src/components/slam.vue:58]: Is this still used for the 3D view? Should we revert this back to GetMap as well?

**Review by tessavitabile — CHANGES_REQUESTED** [review]
Question about reverting back 3D changes.

Also, I haven't yet heard back from Fahmina on whether to only allow a long polling interval for 3D or just remove it.

**Review by tessavitabile — COMMENTED** [review]
I asked Fahmina if she objects to removing 3D for now, and she does not object.

File: web/frontend/src/components/slam.vue (hunk 1 above)
Comment (nicksanford) [reply web/frontend/src/components/slam.vue:58]: leaving this as is as fahimna is good with just not rendering the 3d view for now.

**Review by JohnN193 — APPROVED** [review]

**Review by tessavitabile — APPROVED** [review]

**Review by micheal-parks — APPROVED** [review]

**Comment by micheal-parks** [comment]
@nicksanford FYI you'll need to bump the version in `web/frontend/package.json` to publish the UI to NPM.

**Comment by nicksanford** [comment]
@micheal-parks The slam team is going to hold off on merging this until monday when we can know how to get it on to app.viam.com.

**Comment by DTCurrie** [comment]
Don’t forget to run “npm install” in web/frontend to regenerate the “package-lock.json” file.

**Pull request merged**